private/buf/cmd/buf/command/alpha/plugin/pluginpush/pluginpush.go

```diff
@@ -228,8 +230,8 @@ func run(
 			return err
 		}
 		defer func() {
-			if err := image.Close(); !errors.Is(err, os.ErrClosed) {
-				retErr = multierr.Append(retErr, err)
+			if err := image.Close(); err != nil && !errors.Is(err, storage.ErrClosed) {
```

Comment:
This was the fix for the `already closed` error I encountered with local testing.

Reply:
Any way to test for this?

Reply:
Not easily without wiring up a fair amount of supporting services. To run the CLI it would need to spin up the dependent BSR services and also a Docker client. This code path is less used now that we have implemented plugin sync in the BSR.